app/helpers/heatmap_helper.rb

```diff
@@ -1,61 +1,25 @@
+# This is a class of static helper methods for generating data for the heatmap
+# visualization. See HeatmapHelperTest.
+# See selectedOptions in SamplesHeatmapView for client-side defaults, and
+# heatmap action in VisualizationsController.
 module HeatmapHelper
   DEFAULT_MAX_NUM_TAXONS = 30
+  # Zscore is best for heatmaps because it weighs the frequency against the background
+  DEFAULT_TAXON_SORT_PARAM = 'highest_nt_zscore'.freeze
```

Comment:
Is this new?

Reply:
yes, this was my reasoning and it could be wrong -- thanks for asking. @cdebourcy told me to remove the aggregate score but he didn't say what should replace it as a default.

Reply:
I belive R and/or RPMs are the best since the absolute value of the difference is also relevant. The problem with Z is that it relies on an (often) invalid distribution and loses precision quickly (lots of entries close max value 100)